tests/net/socket/udp/testcase.yaml

```diff
@@ -1,5 +1,5 @@
 tests:
   - test:
-      build_only: true
+      extra_args: CONF_FILE="prj.conf prj-sanitycheck.conf"
```

Comment:
you can avoid this with:

```
extra_configs:
 - CONFIG_NET_TEST=y
 - CONFIG_NET_LOOPBACK=y

```

Then you will not need the `prj-sanitycheck.conf` file

Reply:
this replaces extra_args here

Reply:
Nice hack, will update.

(Testing how it actually runs in CI so far).